### ISSUE TITLE
set height of place that show animation

### ISSUE THREAD

**Issue by farokhze** [gonuit/flutter-custom-refresh-indicator#10]
Hello, Is it possible to set height of box that i show indicator?

**Comment by gonuit** [comment]
### Hi @farokhze 👋🏻 !
Basically, it is possible to do whatever you want (talking about a single indicator 😅).
  
### Please take a look at the [SOURCE CODE](https://github.com/gonuit/flutter-custom-refresh-indicator/blob/master/example/lib/indicators/check_mark_indicator.dart) of this indicator:
![indicator_with_complete_state](https://user-images.githubusercontent.com/37882650/98953759-5f5a2080-24fd-11eb-8c45-0c914aaaac73.gif)

There you have a [_indicatorSize](https://github.com/gonuit/flutter-custom-refresh-indicator/blob/20a2d953f9c4ae8f599093d62f53d99bdc047b0a/example/lib/indicators/check_mark_indicator.dart#L19) property that indicates it's height. You can change it to whatever you want.
Good to know: `IndicatorCntroller` value can change from 0.0 to 1.0 (armed) and then it is possible to over-drag this value to 1.5. So in the above example, the maximum box size will be `_indicatorSize * 1.5`. 

### For a better understanding of `IndicatorController` take a look at this:
![controller_data](https://user-images.githubusercontent.com/37882650/98954058-afd17e00-24fd-11eb-8312-ce4d20651ea3.gif)


### Long story short
This library exposes an `IndicatorController` object that together with `AnimationBuilder` can be used to do any kind of animations/transforms/indicators. 

***

Hope this helps 😃 
Let me know if there's anything else I can help you with 💪🏻 .

**Comment by farokhze** [comment]
Hi thanks it works.
